specifications/connector/radar-altoida-connector.yml

```diff
@@ -0,0 +1,37 @@
+#====================================== Altoida Phone Sensors =====================================#
+vendor: Altoida
+model: altoida.app
+version: 1.0.0
+data:
+  #Phone sensors
+  - doc: Data from the accelerometer sensor of 3 axis in G.
+    topic: altoida_accelerometer
```

Comment:
Perhaps prefix all altoida topics with `upload_` so we keep track of the origin.